changelog/README.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 11th October 2023
+* Extended [Voucher](../operations/vouchers.md#voucher) with new property `Type`
+* Added operations:
+  * [Add vouchers](../operations/vouchers.md#add-vouchers)
+  * [Update vouchers](../operations/vouchers.md#update-vouchers)
+  * [Delete vouchers](../operations/vouchers.md#delete-vouchers)
+  * [Get all voucher codes](../operations/vouchercodes.md#get-all-voucher-codes)
+  * [Add voucher codes](../operations/vouchercodes.md#add-voucher-codes)
+  * [Delete voucher codes](../operations/vouchercodes.md#delete-voucher-codes)
```

Comment:
@MikeAdamsMews  These new operations have a check that in order to work with them, the enterprise must have vouchers enabled in their subscriptions. Should we mention it or is it obvious, that you can't work with something which property don't have in their package? 🤔

Reply:
With all these new endpoints being added, I think I should write a new use case for working with vouchers. In that case, I would put this caveat in the use case. My understanding is that customers on the _Starter_ subscription do not have access to the feature.

Reply:
I'm not sure how subscription etc. works and if its like a package or what🤔 I will leave this probably to you how to describe it? 🤔

Reply:
From the discussion in the standup, the consensus is that it doesn't need a use case and doesn't need a description of the subscription limitation, the reason being that there are lots of features in the API that are dependent on property configuration/subscription and we can't reasonably describe them all. I will keep this under review, but for now no change is needed.